### ISSUE TITLE
Small Bug in Light and Dark theme 

### ISSUE THREAD

**Issue by harshay97** [hardy1334/ip_proj-#3]
I think there is some small bug in light and dark theme,it doesn't change with click **eventlistener** but with **mouseover** eventlistener.I guess this thing needs to be changed.

![Screenshot from 2019-04-22 16-08-50](https://user-images.githubusercontent.com/44117346/56496981-58358100-6519-11e9-814b-346175012ce7.png)

**Comment by harshay97** [comment]
@hardy1334 can u guide me a bit how to work around it ?

**Comment by hardy1334** [comment]
sure, I guess initially u need to be familiar with concept of props in react, just work around it and I guess u will figure it out

**Comment by harshay97** [comment]
can you guide me with some help regarding props in react

**Comment by hardy1334** [comment]
sure you can go through https://reactjs.org/docs/components-and-props.html to get idea about props

**Comment by harshay97** [comment]
Thanks a lot

**Comment by harshay97** [comment]
Is the website compatible with other browsers? such as Chrome and Mozila Firefox?

**Comment by hardy1334** [comment]
yeah,it works on all browsers, the thing is that there some properties of css which works in chrome and may not work in firefox

**Comment by harshay97** [comment]
Okay, I tried to open the same website on a safari browser on a mobile application but seemed to have a totally different layout compared to chrome on an android.

Is this something you would like to fix or is it not a problem?

**Comment by hardy1334** [comment]
this project is currently not responsive(that means ) it looks good on all devices whether its ipad,mobile,pc,etc.

If u want u make it responsive and submit pull request,I will review that

**Comment by harshay97** [comment]
okay. For the js files within the components folder, will they affect the output of the website (the CSS functionality)

**Comment by harshay97** [comment]
Would it be ok if some of the features are changed around the website? or would you like it as it is? I could potentially make it sharper

**Comment by hardy1334** [comment]
sure go for it,looking forward for it

**Comment by harshay97** [comment]
Ill see if it can be done, otherwise it will stay the way it is
